Core/Core/Login/LoginStartViewController.swift

```diff
@@ -371,6 +373,7 @@ extension LoginStartViewController: UITableViewDataSource, UITableViewDelegate,
             view.layoutIfNeeded()
             UIView.animate(withDuration: 0.5) {
                 self.previousLoginsBottom?.constant = -self.previousLoginsView.frame.height
+                self.previousLoginsView.alpha = 0
```

Comment:
Sometimes the view doesn't get all the way down, so this makes it go away, even when the frame.height isn't far enough to move.